scripts/pam_swap.py

```diff
@@ -0,0 +1,126 @@
+#! /usr/bin/env python
+# -*- mode: python; coding: utf-8 -*-
+# Copyright 2016 the HERA Collaboration
+# Licensed under the 2-clause BSD license.
+
+"""
+Script to handle swapping of PAMs.
+"""
```

Comment:
I worry about how much logic is in this script and therefore very difficult to test. Could you make it a very thin (almost no if/else logic) wrapper around a function that could be tested?